hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/assumed_roles.md

```diff
@@ -153,6 +153,14 @@ Here are the full set of configuration options.
   </description>
 </property>
 
+<property>
+  <name>fs.s3a.assumed.role.external.id</name>
+  <value />
+  <description>
+    External id for assumed role, it's an optional configuration.
+  </description>
```

Comment:
insert a link to https://aws.amazon.com/blogs/security/how-to-use-external-id-when-granting-access-to-your-aws-resources/

Reply:
done

Reply:
sorry, I mean't just the URL, not an http 'a' element. it'll probably break a parser if that got pasted in anywhere

Reply:
ok